src/app/(sidebar)/smart-contracts/contract-explorer/components/InvokeContractForm.tsx

```diff
@@ -661,50 +665,78 @@ export const InvokeContractForm = ({
         <Box gap="md">
           {renderSchema()}
 
-          <Box gap="sm" direction="row" align="end" justify="end" wrap="wrap">
-            <Box gap="sm" direction="row" align="end" justify="end" wrap="wrap">
-              <Select
-                id="simulate-tx-xdr-format"
-                fieldSize="md"
-                value={xdrFormat}
-                onChange={(e) => {
-                  setXdrFormat(e.target.value as XdrFormatType);
+          <Box
+            gap="sm"
+            direction="row"
+            align="stretch"
+            justify="space-between"
+            wrap="wrap"
+          >
+            <TabView
+              tab1={{
+                id: "json",
+                label: "JSON",
+                content: jsonResult && (
+                  <SimulatedResponse result={jsonResult} />
+                ),
+                isDisabled: !jsonResult,
+              }}
+              tab2={{
+                id: "base64",
+                label: "Base64",
+                content: base64Result && (
+                  <SimulatedResponse result={base64Result} />
+                ),
+                isDisabled: !base64Result,
+              }}
+              activeTabId={xdrFormat}
+              onTabChange={(id) => {
+                setXdrFormat(id);
+                trackEvent(
+                  TrackingEvent.SMART_CONTRACTS_EXPLORER_INVOKE_CONTRACT_SELECTED_XDR_FORMAT,
+                  {
+                    xdrFormat: id,
+                  },
+                );
+              }}
+            />
+            <Box
+              addlClassName="AbsoluteButtons"
+              gap="sm"
+              direction="row"
+              align="stretch"
+              justify="space-between"
+              wrap="wrap"
+            >
+              <Button
+                size="md"
+                variant="tertiary"
+                disabled={isSimulationDisabled()}
+                isLoading={isSimulating}
+                onClick={async () => {
+                  const xdr = await getXdrToSimulate();
+
+                  if (xdr) {
+                    await handleSimulate(xdr);
+                    await handlePrepareTx(xdr);
+                  }
                 }}
               >
-                <option value="base64">XDR Format: Base64</option>
-                <option value="json">XDR Format: JSON</option>
-              </Select>
+                Simulate
+              </Button>
+
+              <Button
+                size="md"
+                variant="secondary"
+                isLoading={isExtensionLoading || isSubmitRpcPending}
+                disabled={isSubmitDisabled}
+                onClick={handleSimulateAndSubmit}
+              >
+                Simulate & Submit
+              </Button>
             </Box>
```

Comment:
We shouldn't use absolute positioning for layout that affects the flow. In this case, the buttons are out of the layout flow and overlap the tabs. A couple of things we could do:
1. Update `TabView` component to accept an optional element to display on the right (and handle wrapping to make sure the layout doesn't break).
2. Don't place the buttons on the same line as tabs. To me, this looks a bit confusing: tabs now seem like buttons. But I agree that this makes the layout more balanced.

<img width="588" height="459" alt="image" src="https://github.com/user-attachments/assets/16bcb979-3cd4-4953-9c88-e3e4bbd8326f" />

Reply:
Thank you for the suggestion. I will remove `absolute` and Minkyeong oked having a tab view in a separate line :)